### DISCUSSION TITLE
Evaluation AMI Sensor not showing in grid

### DISCUSSION THREAD

**Discussion by Fenr1r-g** [Security-Onion-Solutions/securityonion#8205] [Unsupported Versions]
Hello all, I'm running the SecOnion AMI on a T3a.2xLarge. I followed all the instructions for setup here: https://docs.securityonion.net/en/2.3/cloud-ami.html

I have a sniffing interface setup, and can confirm I am receiving traffic to it (shows suricata/zeek logs, and showing all traffic w/tcpdump -nni eth1), however, no sensor is appearing in the grid display in SOC. See image below.

![image](https://user-images.githubusercontent.com/54542665/176196675-3f9525dc-015f-4f27-b4ea-350e0b54fe79.png)

output for so-status is green across the board and so-test came up with no issues.

**Comment by dougburks** [comment]
Have you checked the logs in `/opt/so/log/` (particularly the `sensoroni` and `soc` subdirectories) for additional clues?
